test/hotspot/jtreg/runtime/valhalla/inlinetypes/UninitializedInlineFieldsTest.java

```diff
@@ -20,42 +20,42 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-package runtime.valhalla.valuetypes;
+package runtime.valhalla.inlinetypes;
 
 import jdk.test.lib.Asserts;
 
 /*
  * @test
  * @summary Uninitialized inline fields test
  * @library /test/lib
- * @compile -XDemitQtypes -XDenableValueTypes -XDallowWithFieldOperator -XDallowFlattenabilityModifiers Point.java JumboValue.java UninitializedValueFieldsTest.java
- * @run main/othervm -Xint -XX:InlineFieldMaxFlatSize=64 runtime.valhalla.valuetypes.UninitializedValueFieldsTest
- * @run main/othervm -Xcomp -XX:InlineFieldMaxFlatSize=64 runtime.valhalla.valuetypes.UninitializedValueFieldsTest
+ * @compile -XDemitQtypes -XDenableValueTypes -XDallowWithFieldOperator -XDallowFlattenabilityModifiers Point.java JumboInline.java UninitializedInlineFieldsTest.java
```

Comment:
Do we still need -XDemitQtypes and -XDenableValueTypes options?

Reply:
The -XDemitQtypes and -XDenableValueTypes options are no longer needed.  I removed them from all the runtime valhalla tests.